tube/etl/__init__.py

```diff
@@ -1,4 +1,3 @@
-import os
 from pyspark import SparkConf, SparkContext
```

Comment:
Should code use the same variables for this Spark: `config.SPARK_EXECUTOR_MEMORY` and `config.SPARK_DRIVER_MEMORY`? It already uses `config.APP_NAME`.